Slack.Automation/Promact.Core.Repository/ExternalLoginRepository/OAuthLoginRepository.cs

```diff
@@ -78,27 +84,29 @@ public OAuthApplication ExternalLoginInformation(string refreshToken)
             var slackUsers = JsonConvert.DeserializeObject<SlackUserResponse>(userDetailsResponse);
             foreach (var user in slackUsers.Members)
             {
-                if (user.Name != StringConstant.SlackBotStringName)
-                {
-                    user.CreatedOn = DateTime.UtcNow;
-                    _slackUserDetails.Insert(user);
-                    _slackUserDetails.Save();
-                }
+                if (!user.Deleted && user.IsBot == false)
```

Comment:
Improve this condition. if you are using a bool property then don't compare it with a bool value.